doc/source/contributing.rst

```diff
@@ -70,6 +70,13 @@ To install additional packages:
 
   pip install sphinx-copybutton
 
+and:
+
+.. code-block:: shell
+  :class: copy-button
+
+  pip install graphviz
```

Comment:
Here I would suggest instead we install the system graphviz. Using pip is not sufficient :)!